src/main/java/org/jenkinsci/account/Application.java

```diff
@@ -1395,7 +1395,7 @@ public AdminUI getAdmin() {
     );
 
     public static final String SPAM_MESSAGE = "The anti-spam system was triggered: we need your help to prove that you are human and not a robot (or a human spammer). " +
-        "Open an issue on <a href='https://github.com/jenkins-infra/helpdesk/issues/new?template=1-report-issue.yml'>the Jenkins Infrastructure Help Desk</a> and provide the email and the userid you tried to register with, as well as the version of Jenkins you currently are running. " +
+        "Open an issue on the Jenkins Infrastructure Help Desk (https://github.com/jenkins-infra/helpdesk/issues/new?template=2-account-issue.yml&amp;labels=triage,accounts&amp;url=account.jenkins.io%2Fantispam) and provide the email and the userid you tried to register with, as well as the version of Jenkins you currently are running. " +
```

Comment:
Cf https://github.com/jenkins-infra/account-app/issues/184 I've removed the HTML for the link.
(no idea how to make [`SystemError`](https://github.com/jenkins-infra/account-app/blob/main/src/main/java/org/jenkinsci/account/Application.java#L220) renders HTML)

Reply:
fixed

<img width="1344" alt="image" src="https://user-images.githubusercontent.com/21194782/203658718-5996ab39-0926-49a3-93d9-c1c9b1199248.png">